egs/swbd/s5c/local/reverberate/run_ivector_common.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash 
+#set -e
+# this script is based on local/nnet3/run_ivector_common.sh
+# but it operates on corrupted training/dev/test data sets
+
+. cmd.sh
+
+stage=1
+snrs="20:10:15:5:0"
+num_data_reps=3
+ali_dir=exp/tri4_ali_nodup_sp
+
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+mkdir -p exp/nnet3_reverberate
```

Comment:
when you move these files to swbd_reverb, just make the experiment directory exp/nnet3. Also you can directly start working on the chain recipe. We need not spend time on the xent versions of the recipe.